Dockerfile

```diff
@@ -73,24 +69,11 @@ RUN apt-get install -y \
       wget \
       xz-utils
 
-# LaTeX -- split to reduce image layer size
-RUN apt-get -y install \
-      texlive-fonts-extra
-RUN apt-get -y install \
-      texlive-latex-extra-doc \
-      texlive-pictures-doc \
-      texlive-publishers-doc
-RUN apt-get -y install \
-      texlive-lang-english \
-      texlive-lang-japanese
-RUN apt-get -y install \
-      texlive-full
-RUN apt-get -y install \
+# Install LateX packages
+RUN export DEBIAN_FRONTEND=noninteractive \
+ && apt-get -y install \
       fonts-symbola \
-      latex-cjk-chinese-arphic-bkai00mp \
-      latex-cjk-chinese-arphic-gbsn00lp \
-      latex-cjk-chinese-arphic-gkai00mp \
-      texlive-fonts-recommended
```

Comment:
Why delete all these latex files?

Reply:
These are actually all transient dependencies of `texlive-full`. I've added the package dependency hull in the commit message of e6ff154184af604dd71fc2e855f29d6dfa900023.

Someone "added" those packages to have smaller, but more layers. Idk, why someone would like to implement such an antipattern.

Reply:
I think this is not 100% true.

Checking https://packages.ubuntu.com/bionic/texlive-full, you will see that, for example, `latex-cjk-chinese-arphic-bkai00mp` is not listed there.

Maybe, one of those dependencies listed there also depends on this one but before removing all of this, we should be sure that we are not removing any package that we had previously installed.